src/csv/CHERI_CSR.csv

```diff
@@ -1,32 +1,32 @@
-"Extended CSR","CLEN Address","Alias","XLEN Address","Mode","Permissions","Reset Value","Action on XLEN write","Action on CLEN write","Executable Vector","Data Pointer","Unseal On Execution","Store full metadata","Zcheri_legacy","Zcheri_purecap","Prerequisites","Description","","","","","","","","","","","","","","","","","","","","",""
-"dpcc","0x7b9","dpc","0x7b1","D","DRW","tag=0, otherwise undefined","Apply <<section_invalid_addr_conv>>.
+"Extended CSR","","Alias","CLEN Address","Mode","Permissions","Reset Value","Action on XLEN write","Action on CLEN write","Executable Vector","Data Pointer","Unseal On Execution","Store full metadata","Zcheri_legacy","Zcheri_purecap","Prerequisites","Description","","","","","","","","","","","","","","","","","","","","",""
+"dpcc","0x7b1","dpc","0x7b1","D","DRW","tag=0, otherwise undefined","Apply <<section_invalid_addr_conv>>.
```

Comment:
Did you make these changes manually in the CSV? We download these CSVs from [this](https://docs.google.com/spreadsheets/d/1nyxKamsYZaeTyG8qP-JX4_oQwcuQ_4nZ_Ihm3RK0NEY/edit?usp=sharing) spreadsheet in the drive because it makes it easier to use, so we normally just change the spreadsheet and do `make download -B`.

Reply:
I'm going to update the spreadsheet and re-commit the CSV